apps/web-e2e/.env.example

```diff
@@ -0,0 +1,6 @@
+# API details
+API_URL="http://localhost:3001"
+API_KEY="123"
+
+# Firebase
+FIREBASE_SERVICE_ACCOUNT={"projectId":"","clientEmail":"","privateKey":""}
```

Comment:
Requires Firebase admin account to be setup, so a user can be added/removed during testing.